index.html

```diff
@@ -641,18 +429,29 @@
 Software developers may want to see the
 <A HREF="https://github.com/expln/metamath-lamp"
 >Metamath-lamp source code repository</A>.
-<LI><I>metamath-exe</I> - the 
+<li><b><i>metamath-exe</i></b> - the 
   <A HREF="mmprog">metamath-exe</A> program is an
   ASCII-based ANSI C program with a command-line interface.
   This is the original program for processing Metamath; today it's
   it's often called "metamath-exe" to distinguish it from other Metamath tools.
-<LI><I>yamma</I> -
+<li><b><i>yamma</i></b> -
   Yamma is a Metamath proof assistant for Visual Studio Code.
   It's implemented as a vSCode extension for Metamath.
```

Comment:
`vSCode`

I think it should be `VS Code`
This page https://code.visualstudio.com/ uses `VS Code` in many places.

Reply:
Thanks!! I'll make that fix.